src/ImageManipulation.php

```diff
@@ -1162,69 +1144,15 @@ public function getAttributes()
                 $attributes['loading'] = 'lazy';
             }
         }
-
-        $attributes = array_merge($attributes, $this->attributes);
-
-        if (isset($attributes['loading']) && $attributes['loading'] === 'eager') {
-            unset($attributes['loading']);
-        }
-
-        $this->extend('updateAttributes', $attributes);
-
         return $attributes;
     }
 
-    /**
-     * Custom attributes to process. Falls back to {@link getAttributes()}.
-     *
-     * If at least one argument is passed as a string, all arguments act as excludes, by name.
-     *
-     * @param array $attributes
-     * @return string
-     */
-    public function getAttributesHTML($attributes = null)
+    public function updateAttributes(array &$attributes): void
     {
-        $exclude = null;
-
-        if (is_string($attributes)) {
-            $exclude = func_get_args();
-        }
-
-        if (!$attributes || is_string($attributes)) {
-            $attributes = $this->getAttributes();
-        }
-
-        $attributes = (array) $attributes;
-
-        $attributes = array_filter($attributes, function ($v) {
-            return ($v || $v === 0 || $v === '0');
-        });
-
-        if ($exclude) {
-            $attributes = array_diff_key(
-                $attributes,
-                array_flip($exclude)
-            );
-        }
-
-        // Create markup
-        $parts = [];
-
-        foreach ($attributes as $name => $value) {
-            if ($value === true) {
-                $value = $name;
-            } else {
-                if (is_scalar($value)) {
-                    $value = (string) $value;
-                } else {
-                    $value = json_encode($value);
-                }
-            }
-
-            $parts[] = sprintf('%s="%s"', Convert::raw2att($name), Convert::raw2att($value));
+        // We don't want an loading=eager attribute because that's the browser default
+        if (isset($attributes['loading']) && $attributes['loading'] === 'eager') {
```

Comment:
This needs to happen after the `$attributes = array_merge($attributes, $this->attributes);` call. This is why its in `updateAttributes`.